packages/components/src/responsive-toolbar-group/README.md

```diff
@@ -19,8 +19,8 @@ import Discover from 'calypso/my-sites/plugins/categories/responsive-toolbar-gro
 
 - `children`[ReactNode]: A group of react components to be rendered.
 - `className`[string]: A classname to add to the ToolBarGroupComponent. (optional).
-- `hideRatio`[number]: The ratio in chich the components are considered to be hidden ([take a look at the IntersectionObserver docs](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#thresholds)) (optional).
-- `showRatio`[number]: The ratio in chich the components are considered to be shown ([take a look at the IntersectionObserver docs](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#thresholds)) (optional).
+- `hideRatio`[number]: The ratio in which the components are considered to be hidden, for example the default ratio of 0.99 will hide the element when 99% of it is visible (1% of it isn't visible). We rely on ([IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#thresholds)) (optional).
+- `showRatio`[number]: The ratio in which the components are considered to be shown, for example the default ratio of 1 will show the element when 100% of it will be visible. We rely on ([IntersectionObserver](https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API#thresholds)) (optional).
```

Comment:
does hiding mean collapsing into 'more'?

Reply:
Yes, the specifics about how it works are in the first few lines of the documentation. Adding it here too would be redundant since you'll usually read the whole documentation (the PR is a special case since you only see the diff)

Reply:
I expanded the diff to read the whole documentation before asking 😆